ledger/catchpointwriter.go

```diff
@@ -128,6 +134,15 @@ func (chunk catchpointFileChunkV6) empty() bool {
 	return len(chunk.Balances) == 0 && len(chunk.KVs) == 0
 }
 
+type catchpointStateProofVerificationData struct {
+	_struct struct{}                                `codec:",omitempty,omitemptyarray"`
+	Data    []ledgercore.StateProofVerificationData `codec:"spd,allocbound=StateProofVerificationDataPerCatchpointFile"`
```

Comment:
Quite small. 80 bytes, one is created every 256 rounds and they only stack if the state proof chain is stuck. For instance, for catchpoints with data to verify 6 months of state proofs, we expect to add 1.4 MB to each catchpoint for the entirety of our data.

Reply:
Around 60 bytes, should grow somewhat after Idan's PR is merged. Since the data is generated every 256 rounds, we expect in the worst case around 1.5 MB per every 6 months of stuck state proofs.